Update fenics legacy instructions (#203)
FEniCS/web: download/archive.md

[Docker CE](https://www.docker.com/products/docker-desktop) for your
platform (Windows, Mac or Linux) and then run the following command:

> You can start a container with the following `docker`
> command:

Reviewer: ```suggestion
```

```bash

> docker run -ti -p 127.0.0.1:8000:8000 -v $(pwd):/root/shared -w /root/shared --name=fenics_legacy ghcr.io/scientificcomputing/fenics-gmsh:2024-05-30

Reviewer: ```suggestion
docker run -ti -p 127.0.0.1:8000:8000 -v $(pwd):/root/shared -w /root/shared ghcr.io/scientificcomputing/fenics-gmsh:2024-05-30
```

```

and resume this container with

```bash
docker container start -i fenics_legacy
```

> at a later instance.

Reviewer: I would just remove this and point readers to other sources for help with Docker.

> For detailed instructions on how to use docker, see for instance the [FEniCS Reference

Reviewer: ```suggestion
For detailed instructions regarding using FEniCS within a docker container, see for instance the [FEniCS Reference
```

Manual](http://fenics-containers.readthedocs.io/en/latest/index.html).

> Note that all reference there to `quay.io`-images should be replaced with the image above.

Reviewer: ```suggestion
Note that all references therein to `quay.io`-images should be substituted with the image above.
```
